<launch>
    <!--joy node-->
    <node respawn="true" pkg="joy" type="joy_node" name="minesweeper_joy">
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.05"/>
    </node>
    <param name="linear_axis" value="1" type="int"/>
    <param name="angular_axis" value="3" type="int"/>
    <param name="linear_scale" value="0.3" type="double"/>
    <param name="angular_scale" value="0.5" type="double"/>

    <node pkg="minesweeper" type="teleop_joy.py" name="minesweeper_teleop"/>
    <node pkg="minesweeper" type="robot_control.py" name="robot_control"/>

</launch>